tutorials/dataframe/distrdf002_dask_connection.py

```diff
@@ -0,0 +1,63 @@
+## \file
+## \ingroup tutorial_dataframe
+## \notebook -draw
+## Configure a Dask connection and fill two histograms distributedly.
+##
+## This tutorial shows the ingredients needed to setup the connection to a Dask
+## cluster (e.g. a `LocalCluster` for a single machine). After this initial
+## setup, an RDataFrame with distributed capabilities is created and connected
+## to a Dask `Client` instance. Finally, a couple of histograms are drawn from
+## the created columns in the dataset. Relevant documentation can be found at
+## http://distributed.dask.org/en/stable .
+##
+## \macro_code
+## \macro_image
+##
+## \date February 2022
+## \author Vincenzo Eduardo Padulano
+from dask.distributed import LocalCluster, Client
+import ROOT
+
+# Point RDataFrame calls to Dask RDataFrame object
+RDataFrame = ROOT.RDF.Experimental.Distributed.Dask.RDataFrame
+
+# Setup connection to a Dask cluster. Two ingredients are needed:
+# 1. Creating a cluster object that represents computing resources. This can be
+#    done in various ways depending on the type of resources at disposal. To use
+#    only the local machine (e.g. your laptop), a `LocalCluster` object cam be
+#    used. This step is optional if the cluster resources are setup externally.
```

Comment:
> This step is optional if the cluster resources are setup externally.

I am worried users will have no idea what that means

Reply:
what about
` This step is optional if the cluster resources are managed by an administrator and they provided you with a URL to connect to the Dask scheduler` ?

Reply:
or `This step can be skipped if you have access to an existing Dask cluster; in that case, the cluster administrator should provide you with a URL to connect to such cluster in Step 2.`